app/views/studios/show.html.erb

```diff
@@ -1,47 +1,48 @@
 <div class="container">
   <h2>Show studio dashboard</h2>
-
-  <br> <h3><b>Name:</b> <%= @studio.name %> </h3>
-  <br> <h4><b>Address:</b> <%= @studio.address %> </h4>
-  <br> <h5><b>Phone:</b> <%= @studio.phone_number %> </h5>
-  <br> <h5><b>Email:</b> <%= @studio.email %> </h5>
-  <br> <h5><b>Webpage:</b> <%= @studio.homepage %> </h5>
-  <%= link_to "Change Studio details", edit_studio_path(current_user.studio_id)%>
+    <br> <h3><b>Name:</b> <%= @studio.name %> </h3>
+    <br> <h4><b>Address:</b> <%= @studio.address %> </h4>
+    <br> <h5><b>Phone:</b> <%= @studio.phone_number %> </h5>
+    <br> <h5><b>Email:</b> <%= @studio.email %> </h5>
+    <br> <h5><b>Webpage:</b> <%= @studio.homepage %> </h5>
+    <%= link_to "Change Studio details", edit_studio_path(current_user.studio_id)%>
 
   <hr>
-  <%= link_to "See all lessons", studio_lessons_path(@studio), class: "btn btn-outline-primary btn-sm" %>
+    <%= link_to "See all lessons", studio_lessons_path(@studio), class: "btn btn-outline-primary btn-sm" %>
   <hr>
+    <!-- can the two days be displayed in the simple calender with the option of scrolling to future dates? -->
+    <%#= calendar number_of_days: 1 do |date| %>
+    <%# end %>
   <h3>List of todays lessons - <%= Date.today.to_formatted_s(:short) %></h3>
-  <% @lessons = Lesson.where(start_date: Date.today) %>
-  <% @lessons.each do |l| %>
-  <p><b>Name: </b>        <%= link_to l.name, lesson_path(l.id) %></p>
-  <p><b>Room: </b>        <%= l.location %></p>
-  <p><b>Description: </b> <%= l.content %></p>
-  <p><b>Places: </b>      <%= l.slots %></p>
-  <hr>
-  <% end %>
+    <% @lessons = Lesson.where(start_date: Date.today) %>
+      <% @lessons.each do |l| %>
+        <p><b>Name: </b>        <%= link_to l.name, lesson_path(l.id) %></p>
+        <p><b>Room: </b>        <%= l.location %></p>
+        <p><b>Description: </b> <%= l.content %></p>
+        <p><b>Places: </b>      <%= l.slots %></p>
+        <hr>
+      <% end %>
 
   <h3>List of tomorrows lessons - <%= Date.tomorrow.to_formatted_s(:short) %></h3>
```

Comment:
Same here: we should add if statements to only display the headers if there are actually any lessons today / tomorrow.